### DISCUSSION TITLE
Welcome to Discussions! Please read this before posting.

### DISCUSSION THREAD

**Discussion by pallupz** [pallupz/covid-vaccine-booking#141] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

Some quick updates:

- I won't be addressing anymore issues for a while - which is why the section was removed.
- To enter the captcha, you may have to close the image first. It is executing synchronously now.
- There are a few pull requests that are in the pipeline. Those will be tested and promoted to main soon.

**Comment by pallupz** [comment]
It'd be best to use Discord for discussions. Please ping either of the below IDs to get added:

- pallupz#5726, 
- sbstratos79#9863 , 
- vishalv26#6244,
- blueclipper#1402, 
- SayantanRC#2099, or 
- sriramcu#8605

**Reply by Ipsi-SG** [reply]
The code is getting timeout in 2/3 mins with below error:-
Unable to Generate OTP
403 <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML><HEAD><META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=iso-8859-1">
<TITLE>ERROR: The request could not be satisfied</TITLE>
</HEAD><BODY>
<H1>403 ERROR</H1>
<H2>The request could not be satisfied.</H2>
<HR noshade size="1px">
Request blocked.
We can't connect to the server for this app or website at this time. There might be too much traffic or a configuration error. Try again later, or contact the app or website owner.
<BR clear="all">
If you provide content to customers through CloudFront, you can find steps to troubleshoot and help prevent this error by reviewing the CloudFront documentation.
<BR clear="all">
<HR noshade size="1px">
<PRE>
Generated by cloudfront (CloudFront)
Request ID: iqjClefWYT2i5RJx0p_nn4oN6SSHysaeaM_-G9SkPUj45AZ-GCODNw==
</PRE>
<ADDRESS>
</ADDRESS>

I would like to understand it is common error or with my configuration.
can anyone help please

**Comment by bhargav1234567890** [comment]
today time out requst id issue in otp

**Comment by Charles4788** [comment]
I have also same error if have solution please reply.

[OneVanilla](https://www.onevanilla.top/)
